kernel/src/Boot/Patches/Patches672.cpp

```diff
@@ -13,48 +17,49 @@ void Mira::Boot::Patches::install_prerunPatches_672()
 	// Use "kmem" for all patches
 	uint8_t *kmem;
 
-	// Patch dynlib_dlsym
-	kmem = (uint8_t*)&gKernelBase[0x1D895A];
-	kmem[0] = 0xE9;
-	kmem[1] = 0xC7;
-	kmem[2] = 0x01;
-	kmem[3] = 0x00;
-	kmem[4] = 0x00;
+	// Enable UART
+	kmem = (uint8_t *)&gKernelBase[0x01A6EB18];
+	kmem[0] = 0x00;
 
-	// Patch a function called by dynlib_dlsym
-	kmem = (uint8_t*)&gKernelBase[0x0041A2D0];
-	kmem[0] = 0x31; // xor eax, eax
-	kmem[1] = 0xC0;
-	kmem[2] = 0xC3;	// ret
+	// Verbose Panics
+	kmem = (uint8_t *)&gKernelBase[0x002ED33A];
+	kmem[0] = 0x90;
+	kmem[1] = 0x90;
+	kmem[2] = 0x90;
+	kmem[3] = 0x90;
+	kmem[4] = 0x90;
 
-	// Patch sys_mmap
-	kmem = (uint8_t*)&gKernelBase[0x000AB57A];
-	kmem[0] = 0x37; // mov     [rbp+var_61], 33h ; '3'
-	kmem[3] = 0x37; // mov     sil, 33h ; '3'
+	// sceSblACMgrIsAllowedSystemLevelDebugging
+	kmem = (uint8_t *)&gKernelBase[0x00233430];
+	kmem[0] = 0xB8;
+	kmem[1] = 0x01;
+	kmem[2] = 0x00;
+	kmem[3] = 0x00;
+	kmem[4] = 0x00;
+	kmem[5] = 0xC3;
 
-	// patch sys_setuid
-	kmem = (uint8_t*)&gKernelBase[0x0010BED0]; // call    priv_check_cred; overwrite with mov eax, 0
-	kmem[0] = 0xB8; // mov eax, 0
-	kmem[1] = 0x00;
+	kmem = (uint8_t *)&gKernelBase[0x00233BD0];
+	kmem[0] = 0xB8;
+	kmem[1] = 0x01;
 	kmem[2] = 0x00;
 	kmem[3] = 0x00;
 	kmem[4] = 0x00;
+	kmem[5] = 0xC3;
 
-	// patch sys_mprotect
-	kmem = (uint8_t*)&gKernelBase[0x00451DB8]; // jnz     loc_FFFFFFFF82652426; nop it out
-	kmem[0] = 0x90;
-	kmem[1] = 0x90;
-	kmem[2] = 0x90;
-	kmem[3] = 0x90;
-	kmem[4] = 0x90;
-	kmem[5] = 0x90;
+	kmem = (uint8_t *)&gKernelBase[0x00233BF0];
+	kmem[0] = 0xB8;
+	kmem[1] = 0x01;
+	kmem[2] = 0x00;
+	kmem[3] = 0x00;
+	kmem[4] = 0x00;
+	kmem[5] = 0xC3;
 
-	// Enable rwx mapping in kmem_alloc
+	// Enable rwx mapping
```

Comment:
Why were the comments removed? Patches are needed in 2 places, kmem_alloc (for kernel/miraloader) and mmap (for user)

Reply:
For every other firmware it has not been included in Mira but in the exploit itself, it's now included in the loader. If this should be kept here it should be added to previous firmware as well, let me know